app/Http/ViewHelpers/Adminland/AdminUploadEmployeeViewHelper.php

```diff
@@ -104,7 +104,7 @@ public static function show(ImportJob $importJob, Employee $loggedEmployee): ?ar
                     'employee' => $importJob->author,
                 ]),
             ],
-            'status' => trans('account.import_employees_status_'.$importJob->status),
+            'status' => $importJob->status,
```

Comment:
Why?

A View Helper is there to reduce the intelligence in the view itself. I like to provide the view with the strings already translated, instead of having to translate it in the view, when we can avoid it.

Reply:
Why: because it must *not* be translated here.
The value is used [there](https://github.com/officelifehq/officelife/blob/4213398b67bd0f1653d48e45ffce173db998797f/resources/js/Pages/Adminland/Employee/Archives/Show.vue#L67), and on other location on the file.

Reply:
@asbiin I've used this in a lot of view helpers. Why should it not be translated in the view helper? It's less logic to the view, so it's better.

Reply:
Again: if you translate it, the view is not working, because the test "report.status == 'uploaded'" will not work in that case !!! I wish the page can work in other languages than English ;)
And it's totally safe to use something like `$t('account.import_employees_status_'+status)` in the view when needed